pkgs/development/libraries/libgtop/default.nix

```diff
@@ -26,6 +27,7 @@ stdenv.mkDerivation rec {
     perl
     gettext
     gobject-introspection
+    fake-uname
```

Comment:
There needs to be a comment that this is to ensure reproducibility.

Reply:
Maybe `fake-uname` should be called `deterministic-uname`.

Reply:
chose `deterministic-uname`